### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by Tamunokuro**
- Create an animals table
- Insert data into the animal's table
- Update queries for the database
query1
query2

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT NOT NULL,
+    neutered BOOLEAN NOT NULL, 
+    weight_kg DECIMAL NOT NULL,
 );
```
Comment (TadesseDev) [inline schema.sql]: - [Optional] Dear @Tamunokuro, Good job creating the table. but I suggest you make the id auto increment IDENTITY(1,1). 

```sql
id int NOT NULL IDENTITY(1,1)
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name from animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (TadesseDev) [inline queries.sql]: - [ ] Dear @Tamunokuro, here we are asked to list the name of all animals born between 2016 and 2019. which means we need to find `date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12'` inclusive. Please update your query accordingly.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Tamunokuro 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (rloterh) [reply queries.sql]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Joshua,  

Your project is complete!🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Table queries properly implemented!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly ensure that your ReadMe for this project adhere(s) to the following standard(s):
   - All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the _FAQ_ section is optional.
   - The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.
   - Kindly ensure that provided text instructions are removed and replaced with appropriate contents applicable under each section of your ReadMe. Similarly, kindly also remove guide notes for _license_ creation under the _license_ section.

Cheers..Happy coding!💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**